client/ayon_blender/api/lib.py

```diff
@@ -311,24 +311,42 @@ def get_selected_collections():
     return [id for id in ids if isinstance(id, bpy.types.Collection)]
 
 
-def get_selection(include_collections: bool = False) -> List[bpy.types.Object]:
+def get_selection(
+        include_collections: bool = False,
+        selected_hierarchies: bool = False
```

Comment:
I think adding this here is an odd choice.

It's a function that gets the selection - the hierarchy below it.. is not your selection? :)
I think we're better of just having a separate `get_hierarchy` function (also see comment below)